components/Navbar/Navbar-styles.js

```diff
@@ -26,10 +24,11 @@ export const NavButton = styled(Link)`
   align-items: center;
   justify-content: center;
   padding: 20px;
-
+  background-color: white;
   margin-left: 30px;
   margin-right: 30px;
 
+  with: 90%;
```

Comment:
```suggestion
  width: 90%;
```

Reply:
And below: I don't know the css properties `icon-align: center`. Do they have any effect?